mixins/form.jsx

```diff
@@ -193,7 +193,7 @@ module.exports = {
     if (errorType === "card_error" && cardError) {
       form.error(cardError.field, cardError.message);
     } else {
-      form.error("other", this.getIntlMessage('try_again_later'));
+      form.error("other", this.getIntlMessage('try_again_later') + " [" + errorType + "]");
```

Comment:
I also understand we won't get many of these screenshots, but, we only need a couple to fix it for the rest.